README.md

```diff
@@ -14,11 +14,21 @@ Bump new version number in:
 
 Commit, tag and create a new release:
 ```shell
-git add README.md lib/docker_cleaner/version.rb
-git commit -m "release: Bump v0.6.0"
-git tag v0.6.0
-git push origin master v0.6.0
-gh release create v0.6.0
+version="0.6.0"
+
+git switch --create release/${version}
+git add Gemfile.lock README.md lib/docker_cleaner/version.rb
```

Comment:
I don't the changes from `lib/docker_cleaner/version.rb` in this PR. Is it normal?

Reply:
Yes I kind of fucked up this release because `master` wasn't protected. It is already updated on the `master` branch :/